### ISSUE TITLE
Tooltips are not persistent

### ISSUE THREAD

**Issue by mshubat** [ableplayer/ableplayer#524]
https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus#persistent

> Once it appears, the content should remain visible until:
> The user removes hover or focus from the trigger and the additional content, consistent with the typical user experience;
> The user dismisses the additional content via the mechanism provided to satisfy the Dismissable condition; or
> The information conveyed by the additional content becomes invalid, such as a 'busy' message that is no longer valid.

![tooltip-dissapears](https://user-images.githubusercontent.com/10766136/172911271-b018820f-ef30-460d-8e36-6b72e6ecfb22.gif)

**Comment by terrill** [comment]
Closed with 6e3899d0f8841d0e4f19738dff2219dc8dc07935

**Comment by terrill** [comment]
@mshubat I had to roll back this fix in 38e1dff3ba7471c3c1bf0af62f87de0b7430b291 

Unfortunately keeping the tooltips visible if a user hovers over them has an undesirable side effect. Since they're positioned over the seekbar, their persistence blocks users from being able to move their pointer from a button to the seekbar.  One possible solution might would be to position the tooltips below the buttons rather than above them. I've reopened this issue for now, while I explore this further.
